source/_integrations/alarm_control_panel.mqtt.markdown

```diff
@@ -244,6 +244,11 @@ state_topic:
   description: The MQTT topic subscribed to receive state updates.
   required: true
   type: string
+supported_features
+  description: A list of features that the alarm control panel supports. The available options are "arm_home", "arm_away", "arm_night", "arm_vacation", "arm_custom_bypass", and "trigger".
+  required: true
+  type: [list, string]
```

Comment:
```suggestion
  type: list
```

Reply:
I erroneously thought `[list, string]` meant "a list of strings". Definitely not what I meant :) Thanks!